crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -2127,23 +2116,16 @@ fn post_greater_than() {
     uv_snapshot!(filters, command(&context)
         .arg("post-greater-than-a>1.2.3")
         , @r###"
-    success: true
-    exit_code: 0
+    success: false
+    exit_code: 1
     ----- stdout -----
 
     ----- stderr -----
-    Resolved 1 package in [TIME]
-    Downloaded 1 package in [TIME]
-    Installed 1 package in [TIME]
-     + package-a==1.2.3.post1
+      × No solution found when resolving dependencies:
+      ╰─▶ Because only package-a<=1.2.3 is available and you require package-a>1.2.3, we can conclude that the requirements are unsatisfiable.
```

Comment:
Similarly, I wonder if we should add a hint that `1.2.3.post1` is available. I think it's probably unnecessary here since `post` releases feel equivalent to the version they follow and are generally transparent to users?